opentelemetry-otlp/src/exporter/http/mod.rs

```diff
@@ -4,6 +4,8 @@ use super::{
 };
 use crate::{ExportConfig, Protocol, OTEL_EXPORTER_OTLP_ENDPOINT, OTEL_EXPORTER_OTLP_HEADERS};
 use http::{HeaderName, HeaderValue, Uri};
+#[cfg(feature = "http-json")]
```

Comment:
shouldn't this be `internal-logs` feature?

Reply:
no, this is only used when json serialization failed, and it was complaining about unused imports without this!